modules/sdk-core/src/bitgo/wallet/wallet.ts

```diff
@@ -2723,6 +2727,7 @@ export class Wallet implements IWallet {
             lowFeeTxid: params.lowFeeTxid,
             receiveAddress: params.receiveAddress,
             feeOptions,
+            isRebuild: !!params.isRebuild,
```

Comment:
Also false for acceleration since we are trying to accelerate the existing transaction and no need to consider pending approvals?